src/InventoryManager.java

```diff
@@ -0,0 +1,263 @@
+                    if(!itemMap.containsKey(name)) {
+                        System.out.println(name + " Does not exist.");
+                        continue;
+                    }
+                    Inventory inventory = itemMap.get(name);
+                    int i = inventory.getQuantity();
+                    if(Integer.parseInt(quantity) > i) {
+                        throw new Exception("Inventory Quantity less than " + quantity);
+                    }
+                    else {
+                        inventory.setQuantity(inventory.getQuantity() - Integer.parseInt(quantity));
+                    }
+                }
+
+                // remove item from map
+                else if(option.equals("3")) {
+                    System.out.println("Enter name");
+                    String line = br.readLine();
+                    String[] parts = line.split(",");
+                    if((parts == null) || (parts.length != 1)) {
+                        System.out.println("Invalid name");
+                        continue;
+                    }
+                    String name = parts[0];
+
+                    if(!itemMap.containsKey(name)) {
+                        System.out.println(name + " Does not exist.");
+                        continue;
+                    }
+                    itemMap.remove(name);
+                }
+
+                // update item quanity
+                else if(option.equals("4")) {
+                    if(itemMap.size() <= 0) {
+                        System.out.println("No items found.");
+                        continue;
+                    }
+                    System.out.println("Choose from items below by name");
+                    Iterator iter = itemMap.keySet().iterator();
+                    while(iter.hasNext()) {
+                        System.out.println((String)iter.next());
+                    }
+                    String name = br.readLine();
+                    if(!itemMap.containsKey(name)) {
+                        System.out.println("Invalid item " + name);
+                        continue;
+                    }
+                    System.out.println("Enter new quantity: ");
+                    String quantity = br.readLine();
+                    if(!isInteger(quantity)) {
+                        System.out.println("Invalid Quantity " + quantity);
+                        continue;
+                    }
+                    Inventory inventory = itemMap.get(name);
+                    inventory.setQuantity(Integer.parseInt(quantity));
+                }
+
+                // update item price
+                else if(option.equals("5")) {
+                    if(itemMap.size() <= 0) {
+                        System.out.println("No items found.");
+                        continue;
+                    }
+                    System.out.println("Choose from items below by name");
+                    Iterator iter = itemMap.keySet().iterator();
```

Comment:
This is another thing that we haven't gone over.  Why use this instead of a simple for loop?